src/lib/components/PosPaymentsList.svelte

```diff
@@ -0,0 +1,76 @@
+<script lang="ts">
+	import PriceTag from '$lib/components/PriceTag.svelte';
+	import type { Currency } from '$lib/types/Currency';
+	import { useI18n } from '$lib/i18n';
+
+	const { t } = useI18n();
+
+	export let payments: Array<{
+		number: number;
```

Comment:
I think paymentId is more useful than a sequential number. This is already a sorted list and we can use the createdAt as as natural sorting key.